.travis.yml

```diff
@@ -6,6 +6,7 @@ before_script:
 script:
   - npm test
   - gulp build-prod
+  - bash publish.sh
```

Comment:
are there no packages available which takes care of publishing from e.g. gulp. 

Just wondering why we did have to write this on our own.

Reply:
> I've based this on existing code from us and this seems to be very reliable

That's why ☝️ I'd like to get this step automated with minimal effort. I'm sure there are packages available around this though and we can definitely improve it in the future if needed.

Reply:
@offnertho ok for you to move forward?